### ISSUE TITLE
Thread on single line with single-argument functions

### ISSUE THREAD

**Issue by MicahElliott** [bbatsov/clojure-style-guide#206]
When threading a few calls single-argument calls that will fit within 80 characters, prefer a single line.

```clojure
;; good
(-> x fizz :foo first frob)

;; not as good
(-> x
    fizz
    :foo
    first
    frob)
```

But when one of more functions takes arguments, split each call onto a separate line.

```clojure
;; good
(-> x
    (fizz a b)
    :foo
    first
    (frob x y))

;; not as good
(-> x (fizz a b) :foo first (frob x y))
```

**Comment by bbatsov** [comment]
I agree with the general premise, although I guess that's a bit subjective. Still, I wouldn't mind if we mention those suggestions in the guide.

**Comment by danielcompton** [comment]
This seems pretty subjective to me and pretty hard to make any strict recommendations. Often I find that I'll split a threading macro onto multiple lines if I want to really emphasis what is happening, and leave it on a single line if it's a trivial transformation, even if there is a functional call with arguments.

**Comment by bbatsov** [comment]
Same here.
